python/triton/compiler/compiler.py

```diff
@@ -72,6 +74,9 @@ def optimize_ttgir(mod, num_stages, arch):
     pm.add_tritongpu_remove_layout_conversions_pass()
     if _is_cuda(arch):
         pm.add_tritongpu_accelerate_matmul_pass(arch)
+    # TODO change interface of accelerate_matmul_pass
+    if torch.version.hip is not None and gpu_has_mfma():
```

Comment:
Maybe move `if torch.version.hip` check inside `gpu_has_mfma` function?

Reply:
Added the check to `gpu_has_mfma`, but left it also here to mark the places where the fork differs from upstream.